src/um/http.rs

```diff
@@ -851,7 +860,7 @@ extern "system" {
         pReserved: PVOID,
     ) -> ULONG;
     pub fn HttpCreateHttpHandle(
-        pReqQueueHandle: HANDLE,
+        pReqQueueHandle: PHANDLE,
```

Comment:
Even though this is theoretically a breaking change, it would actually be impossible to be correctly using this API as it was without crashing. So looks good to me 👍

Reply:
Breaking changes are allowed when they are correcting bindings.

Reply:
Cool, then my approval of it stands, looks good to merge~